### DISCUSSION TITLE
Small Feature Map

### DISCUSSION THREAD

**Discussion by PhenomenalOnee** [STVIR/pysot#385] [General] [answered]
Nice work. I am thinking to train this using lighter bacbone network like tiny darknet. The problem is the output feature map from tiny darknet is of size (128,15,15) for input (3,255,255) and size (128,7,7) for input (3,127,127). So now after DepthwiseRPN, i get feature map whose size is (10/20,9,9). As anchor generation depends on this feature map size, so total number of anchors here will be 5x9x9 i.e 405 which is relatively less compared to alexnet (5x17x17 , 1445). So will this impact the regression for bbox output or is there a way to use smaller feature map size so that lesser anchors can be used.

**Comment by ZhiyuanChen** [comment]
It really depends on your search region. If your search region is not too big, I believe the reduce of number of anchors should not have a very significant influence.

**Comment by PhenomenalOnee** [comment]
OK so i use search region of let's say 180, it will compensate for lesser anchors. I will give it a go.
Thank you for the replay.

**Comment by ZhiyuanChen** [comment]
> OK so i use search region of let's say 180, it will compensate for lesser anchors. I will give it a go.
> Thank you for the replay.

No worries, sorry for this late reply, and let us know the outcome~

**Comment by PhenomenalOnee** [comment]
One thing, if i try to reduce image search size, than the feature map size will further reduces hence making more less anchors. So   i think, its not going to help in that?
what you prefer @zyc-ai

**Comment by ZhiyuanChen** [comment, marked as answer]
> One thing, if i try to reduce image search size, than the feature map size will further reduces hence making more less anchors. So i think, its not going to help in that?
> what you prefer @zyc-ai

It highly depends on your task as it is actually a hypterparameter. So if your objects move fast, smaller image size and larger search region would have been better, vice versa.
